### PR TITLE
Update area.py

### PR DESCRIPTION
**Pull request by SabariGanesh-K**
Added Area for Rhombhus

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by SabariGanesh-K** [comment]
I have added a new shape for Area. All checks have been passed :)

**Comment by poyea** [comment]
When I type `Area for Rhombus` into Google, it asks for 2 diagonals. I guess this is the more accepted version of it right?

File: maths/area.py [hunk 1]
```diff
@@ -186,6 +186,30 @@ def area_circle(radius: float) -> float:
     return pi * radius ** 2
 
 
+def area_rhombhus(base: float, height: float) -> float:
```
Comment (poyea) [inline maths/area.py:189]:
```suggestion
def area_rhombus(base: float, height: float) -> float:
```

File: maths/area.py [hunk 2]
```diff
@@ -186,6 +186,30 @@ def area_circle(radius: float) -> float:
...
+        ...
+    ValueError: area_rhombhus() only accepts non-negative values
+    >>> area_rhombhus(-1, 2)
+    Traceback (most recent call last):
+        ...
+    ValueError: area_rhombhus() only accepts non-negative values
+    """
+    if base < 0 or height < 0:
+        raise ValueError("area_rhombhus() only accepts non-negative values")
+    return 1 / 2 * base * height
```
Comment (poyea) [inline maths/area.py:210]: Probably make it take two diagonals as parameters.

**Review by poyea — CHANGES_REQUESTED** [review]

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**